vm/src/vm.rs

```diff
@@ -764,8 +758,7 @@ impl VirtualMachine {
         code_object: &bytecode::CodeObject,
```

Comment:
Maybe we can move this to `objfunction.rs` so that we can make the fields all private?

Reply:
I did it. Will you please review newly added commits?

Reply:
I moved a few methods and now it looks better

Reply:
Looks good!